test/test_cuda.py

```diff
@@ -3967,6 +3968,75 @@ def test_lazy_init(self):
         rc = check_output([sys.executable, '-c', test_script]).decode("ascii").strip()
         self.assertEqual(rc, "0")
 
+    def test_event_creation_callback(self):
+        torch._C._activate_cuda_trace()
+        self.mock = unittest.mock.MagicMock()
```

Comment:
Does this need to be `self.mock`? Can't it just be a local variable?

Reply:
I also do wonder now whether these tests are run in "isolation", each of them in a new process, or if they are all run one after the other in the same process? In the latter case, I'm worried that by installing this hook callback but by never removing it, it will fire for all later tests and would slow them down.